t/009-error-log-append.t

```diff
@@ -0,0 +1,171 @@
+use Test::Nginx::Socket::Lua;
+
+#worker_connections(1014);
+#master_process_enabled(1);
+#log_level('warn');
+
+repeat_each(2);
+
+plan tests => repeat_each() * (blocks() * 5) - 2;
+
+#no_diff();
+no_long_string();
+#master_on();
+#workers(2);
+check_accum_error_log();
+run_tests();
+
+__DATA__
+
+
```

Comment:
need to add one test case test whether the request_id changed during rewrite or access phase, may be

```

        lua_kong_error_log_request_id $my_var;
        set $my_var "";

         rewrite_by_lua_block {
         ngx.log(ngx.ERR, "...") -- check whether it is empty
         ngx.var.my_var = ..;      -- modify request_id variable
         ngx.log(ngx.ERR, "...")  --  check whether request_id has been changed
         }

         access_by_lua_block {
           ngx.log(ngx.ERR, "...")  --  check whether request_id has been logged during access phase
         }

```

Reply:
added one test case with what should be all the possible changes the request ID can have